tools/packager/packager.py

```diff
@@ -73,6 +74,8 @@ def run_packager():
     if args.cmd == "help" or not args.cmd:
         print(USAGE)
 
+    elif args.cmd == "version":
+        print("{0}: version {1}".format(DESCRIPTION, VERSION))
```

Comment:
can we encode the git hash in another attribute and do something like:
```Python
version = GITHASH if VERSION.startswith("@") else VERSION
print("{0}: version {1}".format(DESCRIPTION, version ))
```
This way if you run it from source you'll see the hash instead of the template variable.